### PR TITLE
"Image button has non-empty accessible name" (59796f): Use "attribute value"

### LINKED ISSUE #1066: Inapplicable example 2 is confusing in Image button has accessible name (59796f)
Inapplicable Example 2 is confusing in Image button has accessible name.
Similar to applicability, Inapplicable Example 2, is confusing. 

"The input element with type with a type attribute in the Button state is not an image button. Success Criterion 1.1.1 Non-text Content can not fail text buttons. Only non-text content is applicable."

Type is not really a state but rather an attribute. I would simplify to read 

"The input element with type with a type attribute in the Button is not applicable to this rule. The rule [Button has accessible name](https://act-rules.github.io/rules/97a4e1#applicability) applies to Button elements. Success Criterion 1.1.1 Non-text Content can not fail text buttons. Only non-text content is applicable."

### PR DESCRIPTION
**Pull request by Jym77**
Use "attribute value" to speak about the state of the `type` attribute.

Closes issue(s):

- closes #1066

Need for Final Call:
This will require a 1 week Final Call (technically changing Applicability)

---

### **When creating PR:**

- [X] Make sure you're requesting to **pull a branch** (right side) to the `develop` branch (left side).
- [X] Make sure you do not remove the "How to Review and Approve" section in your pull request description

### **After creating PR:**

- [X] Add yourself (and co-authors) as "Assignees" for PR.
- [X] Add label to indicate if it's a `Rule`, `Definition` or `Chore`.
- [X] [Link the PR to any issue it solves](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue). This will be done automatically by [referencing the issue](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) at the top of this comment in the indicated place.
- [X] Optionally request feedback from anyone in particular by assigning them as "Reviewers".

### **When merging a PR:**

- [ ] Close any issue that the PR resolves. This will happen automatically upon merging if the PR was correctly linked to the issue, e.g. by referencing the issue at the top of this comment.

## How to Review And Approve

- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines.
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
- Make sure to also review the proposed Final Call period. In case of disagreement, the longer period wins.

### CODE REVIEW

**Review by kasperisager — APPROVED** [review]

File: _rules/image-button-non-empty-accessible-name-59796f.md [hunk 1]
```diff
@@ -137,7 +137,7 @@ The `button` element is not an image button. [Success Criterion 1.1.1 Non-text C
 
 #### Inapplicable Example 2
 
-The `input` element with type with a `type` attribute in the `Button` state is not an image button. [Success Criterion 1.1.1 Non-text Content](https://www.w3.org/TR/WCAG21/#non-text-content) can not fail text buttons. Only non-text content is applicable.
+The `input` element with type with a `type` [attribute value][] of `button` is not an image button. [Success Criterion 1.1.1 Non-text Content](https://www.w3.org/TR/WCAG21/#non-text-content) can not fail text buttons. Only non-text content is applicable.
```
Comment (ajanec01) [inline _rules/image-button-non-empty-accessible-name-59796f.md:140]:
```suggestion
The `input` element with the `type` [attribute value][] of `button` is not an image button. [Success Criterion 1.1.1 Non-text Content](https://www.w3.org/TR/WCAG21/#non-text-content) can not fail text buttons. Only non-text content is applicable.
```

**Review by ajanec01 — APPROVED** [review]
Minor issue, the "with type" phrase is used twice in one sentence.

File: _rules/image-button-non-empty-accessible-name-59796f.md (hunk 1 above)
Comment (WilcoFiers) [reply _rules/image-button-non-empty-accessible-name-59796f.md:140]: This please.

**Review by WilcoFiers — DISMISSED** [review]
Please use Aron's suggestion.

File: _rules/image-button-non-empty-accessible-name-59796f.md (hunk 1 above)
Comment (Jym77) [reply _rules/image-button-non-empty-accessible-name-59796f.md:140]: 🙈 Good catch

**Review by carlosapaduarte — APPROVED** [review]

**Comment by Jym77** [comment]
Final Call ends on December 18th.

**Comment by Jym77** [comment]
Final Call has ended. Merging.

**Pull request merged**